cmd/k8s-operator/deploy/chart/values.yaml

```diff
@@ -8,6 +8,17 @@ oauth: {}
   # clientId: ""
   # clientSecret: ""
 
+# Secret volume. If set it defines the volume the oauth secrets will be mounted
```

Comment:
Could we also document that:
- the OAuth client creds needs to be provided via `client_id`, `client_secret` keys
- this block would override the `.oauth` block

Reply:
done